packages/symfony-bridge/composer.json

```diff
@@ -25,6 +25,13 @@
             "homepage": "http://php-and-symfony.matthiasnoback.nl"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/greg0ire/ProxyManager.git",
+            "description": "Use fork until PHP 8 is supported https://github.com/Ocramius/ProxyManager/pull/628"
```

Comment:
Can be removed when https://github.com/Ocramius/ProxyManager/pull/628 is merged

Reply:
https://github.com/SimpleBus/SimpleBus/issues/93